mmv1/templates/terraform/examples/container_attached_cluster_basic.tf.erb

```diff
@@ -1,6 +1,11 @@
 data "google_project" "project" {
 }
```

Comment:
I would prefer splitting this down to two PRs.. one for each data source. To keep the context more focused. It doesn't seem like the data sources are dependent on eachother.